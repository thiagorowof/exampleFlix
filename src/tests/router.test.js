import React from 'react';
import renderer from 'react-test-renderer';
import { Link } from 'react-router-dom';
import { StaticRouter } from 'react-router'

test('Verify that the path to the clients screen exists', () => {
  const context = {};
  const component = renderer.create(
    <StaticRouter location="/Clients" context={context}>
      <Link to="/Clients" />
    </StaticRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});


test('Verify that the path to the dashboard screen exists', () => {
  const context = {};
  const component = renderer.create(
    <StaticRouter location="/#" context={context}>
      <Link to="/#" />
    </StaticRouter>
  );

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});