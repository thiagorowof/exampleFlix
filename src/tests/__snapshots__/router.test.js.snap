// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Verify that the path to the clients screen exists 1`] = `
<a
  href="/clientes"
  onClick={[Function]}
/>
`;

exports[`Verify that the path to the dashboard screen exists 1`] = `
<a
  href="/"
  onClick={[Function]}
/>
`;
